vel_motor cmd_vel
opt_rotary Optocoupler
int64 servo
int32 ultrasonic
float32 kecepatan
float32 sudut